manifests/c/CoolPlayLin/LANDrop/0.4.0/CoolPlayLin.LANDrop.installer.yaml

```diff
@@ -3,10 +3,17 @@
 
 PackageIdentifier: CoolPlayLin.LANDrop
 PackageVersion: 0.4.0
+Platform:
+  - Windows.Desktop
+InstallerType: inno
 Installers:
 - Architecture: x64
-  InstallerType: inno
+  Scope: machine
   InstallerUrl: https://github.com/CoolPlayLin/LANDropSetup/releases/download/0.4.0/LANDrop.Setup.ia32.System.exe
   InstallerSha256: F8594706B60138D4202A24C46BDC9E0B5F2389623870A8BC7F823622311BCEED
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://github.com/CoolPlayLin/LANDropSetup/releases/download/0.4.0/LANDrop.Setup.ia32.User.exe
+  InstallerSha256: D364EB7647835A586BD245F5286911E7074F1BBFF4B007891C324B310F9671E2
```

Comment:
I'm seeing this:
![image](https://github.com/microsoft/winget-pkgs/assets/11600822/a8a3e69e-2256-4989-951c-2bf2b13f15cb)
![image](https://github.com/microsoft/winget-pkgs/assets/11600822/9ad85017-2208-403f-a765-88459cf9e9b0)
![image](https://github.com/microsoft/winget-pkgs/assets/11600822/fbd1fc49-bbe3-458b-97c9-111c1abdd4f6)